jobs/JGDAS_ATMOS_GLDAS

```diff
@@ -0,0 +1,136 @@
+#! /usr/bin/env bash
+
+source "${HOMEgfs:?}/ush/preamble.sh"
+
+export RUN_ENVIR=${RUN_ENVIR:-"nco"}
+
+#############################
+# Source relevant config files
+#############################
+export EXPDIR=${EXPDIR:-${HOMEgfs}/parm/config}
+configs="base gldas"
+config_path=${EXPDIR:-${NWROOT}/gfs.${gfs_ver}/parm/config}
+for config in ${configs}; do
+    # shellcheck disable=SC1090-SC1091
+    . "${config_path}/config.${config}"
+    # shellcheck disable=
+    status=$?
+    [[ ${status} -ne 0 ]] && exit "${status}"
+done
+
+
+if [[ "${cyc:?}" -ne "${gldas_cyc:?}" ]]; then
+    echo "GLDAS only runs for ${gldas_cyc} cycle; Skip GLDAS step for cycle ${cyc}"
+    exit 0
+fi
+
+gldas_spinup_hours=${gldas_spinup_hours-:72}
+xtime=$((gldas_spinup_hours+12))
+if [[ "${CDATE}" -le "$(${NDATE:?} +"${xtime}" "${SDATE:?}")" ]]; then
+    echo "GLDAS needs fluxes as forcing from cycles in previous ${xtime} hours"
+    echo "starting from ${SDATE}. This gldas cycle is skipped"
+    exit 0
+fi
+
+##########################################
+# Source machine runtime environment
+##########################################
+# shellcheck disable=SC1090-SC1091
+. "${HOMEgfs}/env/${machine:?}.env" gldas
+# shellcheck disable=
+status=$?
+[[ ${status} -ne 0 ]] && exit "${status}"
+
+
+##############################################
+# Obtain unique process id (pid) and make temp directory
+##############################################
+export pid=${pid:-$$}
+export outid=${outid:-"LL${job}"}
+
+export DATA=${DATA:-${DATAROOT}/${jobid:?}}
+mkdir -p "${DATA}"
+cd "${DATA}" || exit
```

Comment:
This should be a non-zero exit since it is an error.

Reply:
Added "1" with the exit:

```
cd "${DATA}" || exit 1
```